packages/status/src/class-status.php

```diff
@@ -143,6 +143,12 @@ public function is_local_site() {
 			}
 		}
 
+		// @todo Remove function_exists when WP 5.5 is the minimum version.
+		// Use Core's environment check, if available.  Added in 5.5.0 / 5.5.1 (for `local` return value)
+		if ( function_exists( 'wp_get_environment_type' ) && 'local' === wp_get_environment_type() ) {
+			$is_local = true;
```

Comment:
We could move this above the regex above, so if it's true, we don't cycle through the list of local URLs.

Reply:
Self-resolved via 43d11e6. I'll leave it as `needs review` since I made a fairly "large" change to the PR. :)